gulpfile.js

```diff
@@ -56,15 +65,24 @@ gulp.task('copy-static-assets-from-design-system', () => {
     .pipe(gulp.dest(`${OUTPUT_DIRECTORY}/favicons`))
 })
 
+gulp.task('build-styles', () => {
+  return gulp
+    .src('./src/scss/*.scss')
+    .pipe(sassCompiler(sassOptions).on('error', sassCompiler.logError))
+    .pipe(gulp.dest('./dist/assets/css'))
+});
+
 gulp.task('build-script', gulp.series(...scripts.map(createBuildScriptTask)))
 
 gulp.task('watch-and-build', async () => {
     gulp.watch('./src/js/**', gulp.series('build-script'))
+    gulp.watch('./src/scss/*.scss', gulp.series('build-styles'))
 })
 
+
 gulp.task(
   'build',
-  gulp.series('build-script', 'copy-static-assets-from-design-system')
+  gulp.series('build-script', 'build-styles', 'copy-static-assets-from-design-system')
```

Comment:
I think this can be sped up further with the following

```javascript
gulp.series(parallel('build-script', 'build-styles'), 'copy-static-assets-from-design-system')
```